stdlib/2.7/socket.pyi

```diff
@@ -278,7 +278,7 @@ class socket:
     proto = 0
 
     def __init__(self, family: int = ..., type: int = ...,
-                 proto: int = ..., fileno: int = ...) -> None: ...
+                 proto: int = ..., fileno: Optional[int] = ...) -> None: ...
```

Comment:
Good point. So should I remove the argument all together, since it's not
documented, or figure out what the type is?

On Aug 27, 2016 3:30 PM, "Guido van Rossum" <notifications@github.com>
wrote:

> In stdlib/2.7/socket.pyi
> <https://github.com/python/typeshed/pull/504#discussion_r76523203>:
>
> > @@ -278,7 +278,7 @@ class socket:
> >      proto = 0
> >
> >      def __init__(self, family: int = ..., type: int = ...,
> > -                 proto: int = ..., fileno: int = ...) -> None: ...
> > +                 proto: int = ..., fileno: Optional[int] = ...) -> None: ...
>
> Yeah, the _sock argument is not an int, unlike the PY3 file no argument.
> … <#m_4218818303674341740_>
> --Guido (mobile)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/python/typeshed/pull/504/files/231b0cfab31bc75a440669d1796062952c79ecee#r76523203>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCCAkQhxwLlG5Hxz-0lUO9VxN-YZnl3ks5qkLp_gaJpZM4Juw-n>
> .
>